WalletWasabi/WebClients/Wasabi/WasabiClient.cs

```diff
@@ -75,24 +81,31 @@ public async Task<SynchronizeResponse> GetSynchronizeAsync(uint256 bestKnownBloc
 		/// <remarks>
 		/// Throws OperationCancelledException if <paramref name="cancel"/> is set.
 		/// </remarks>
-		public async Task<FiltersResponse> GetFiltersAsync(uint256 bestKnownBlockHash, int count, CancellationToken cancel = default)
+		public async Task<FiltersResponse?> GetFiltersAsync(uint256 bestKnownBlockHash, int count, CancellationToken cancel = default)
```

Comment:
This is because the method returns `null` as some kind of error code. I think methods should not return nulls in most of the cases.

Reply:
`GetFiltersAsync` is used only in tests. Not sure if it is otherwise useful.